pkg/apiserver/registry/system/supportbundle/rest.go

```diff
@@ -249,11 +250,11 @@ func (r *supportBundleREST) collect(ctx context.Context, dumpers ...func(string)
 	}, nil
 }
 
-func (r *supportBundleREST) collectAgent(ctx context.Context) (*systemv1beta1.SupportBundle, error) {
+func (r *supportBundleREST) collectAgent(ctx context.Context, cacheCopy *systemv1beta1.SupportBundle) (*systemv1beta1.SupportBundle, error) {
```

Comment:
`cacheCopy` is not helpful for the argument's meaning, just `supportBundle`?

Reply:
Sure. Updated.